tests/unit_tests/test_get_decorated_nodes.py

```diff
@@ -0,0 +1,81 @@
+import ast
+from typing import Union
+
```

Comment:
nit: remove linebreaks

Reply:
this is actually how `isort` wants it... you can see i tried to remove this [here](https://github.com/MagnivOrg/magniv-core/pull/44/commits/390c0e131f20c35c2bf27c7dc3e3b1205e92c0e6) but it failed GH checks. I then ran the command run in GH actions less the `--check` flag and got [this](https://github.com/MagnivOrg/magniv-core/pull/44/commits/635a26a1979e72485fccf2e48a8c0bc7cde73329)